### ISSUE TITLE
t/anynum.t test failures

### ISSUE THREAD

**Issue by eserte** [trizen/Math-GComplex#1]
On some systems (freebsd 9.2, freebsd 11.0, debian jessie) the test suite fails:
```
#   Failed test at t/anynum.t line 215.
#          got: '2.62550657522387017433921906805091153104285164589 -0.134463574038477498677542285560005086918012804483'
#     expected: '-0.516086078365923064123424315228591353154317753489 -0.134463574038477498677542285560005086918012804483'
```
Observation: the difference of the first component is π

**Comment by eserte** [comment]
I think now that not the operating system is the problem, but a too old version of Math::AnyNum.

**Comment by trizen** [comment]
Hi. Thanks for reporting. There was a minor issue in the `atan2()` function in [Math::AnyNum](https://metacpan.org/release/Math-AnyNum) (fixed in version 0.20), which caused this test to fail.

Installing Math::AnyNum v0.20 should make the test pass.
